src/main.rs

```diff
@@ -1098,6 +1141,11 @@ fn main() {
                         .help( "Generate webasm through Emscripten" )
                         .overrides_with( "target-asmjs-emscripten" )
                 )
+                .arg(
+                    Arg::with_name( "target-webasm-unknown" )
```

Comment:
I think that the `-unknown` postfix is somewhat superflous; since it's (I assume) going to be the default backend in the future I'd prefer to simply call this `target-webasm`.

Reply:
Will be changed in next patch version.